diffprivlib/models/__init__.py

```diff
@@ -24,3 +24,4 @@
 from diffprivlib.models.logistic_regression import LogisticRegression
 from diffprivlib.models.pca import PCA
 from diffprivlib.models.standard_scaler import StandardScaler
+from diffprivlib.models.tree import RandomForestClassifier
```

Comment:
Can we change the filename to `forest.py` to match the equivalent in scikit-learn?